### ISSUE TITLE
Needs hang detection

### ISSUE THREAD

**Issue by MarkAtwood** [awslabs/yesiscan#24]
If a backend cant process a file in 60 seconds, it should give up, emit a warning message, and move on

**Comment by ovalba** [comment]
True, I agree that we need this, but our use case is switching to a more "lite" approach. We will enable just one backend, the rest will be optional. when we enable the others, we will also support pipelines and start/stop aproach
